core/src/main/resources/lib/layout/html.jelly

```diff
@@ -0,0 +1,177 @@
+<!--
```

Comment:
@jglick yes, changing this is dependent on: https://github.com/jenkinsci/jenkins/pull/2029